deposit/src/main/java/edu/unc/lib/deposit/work/DepositSupervisor.java

```diff
@@ -613,6 +611,18 @@ private void queueNextJob(Job job, String depositUUID, Map<String, String> statu
 			depositStatusFactory.setState(depositUUID, DepositState.finished);
 			metricsClient.incrFinishedDeposit();
 			
+			Map<String, String> deposit = depositStatusFactory.get(depositUUID);
```

Comment:
I believe you don't need to retrieve the deposit status again here as it is already being passed into this method in the "status" parameter